multimedia/mlt/Portfile

```diff
@@ -22,11 +24,10 @@ long_description    MLT is an open source multimedia framework, designed and dev
                     This package contains the files necessary for running applications that use \
                     the MLT multimedia framework.
 homepage            http://www.mltframework.org/
-master_sites        sourceforge:project/mlt/mlt
 platforms           darwin
 
-checksums           rmd160  647cdad41974b93c5745a358e1129f2ed5f6e577 \
-                    sha256  d3a992f3e67463e68630cb0b455d408a2a12f4da7a19e46807fa08a79f09b2b6
+checksums           rmd160  5aa9892ea06b2bf0e9abced6d69cf9005f05fa49 \
+                    sha256  54d6c28c8eae23b77c57bc59eec441ac073c56d029f59132cce647a8ace9b0dc
```

Comment:
Just as a note, we are now adding `size` to `rmd160` and `sha256`. If you put in `size 0`, and run `port -v checksum`, it will generate the right lines for you.